### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by tommylettieri**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "@types/bluebird": "^3.5.30",
+    "@types/express": "^4.17.6",
+    "@types/node": "^13.11.1",
+    "@types/validator": "^13.0.0",
+    "sequelize-cli": "^5.5.1",
+    "typescript": "^3.8.3"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npm run build && node ts-built/index.js",
```
Comment (gbhojraj) [inline package.json:24]: I'm assuming this currently requires a manual restart, and we'll add nodemon in later?

File: src/database/index.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+import appSequelize from './app-sequelize';
+
+import user from './models/user';
+
+(async () => {
```
Comment (gbhojraj) [inline src/database/index.ts:6]: Does this need to eventually be `await`ed?

File: package.json (hunk 1 above)
Comment (tommylettieri) [reply package.json]: Yup

Comment (tommylettieri) [reply package.json]: I didn't have a chance to get nodemon, with typescript working, especially getting it to work with stack traces (source maps) currently we do this with ts-node

**Pull request merged**